hack/deploy-test-progsync.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+root=$(dirname "${BASH_SOURCE[0]}")
+
+prevcontext=$(kubectl config current-context)
+kubectl config use-context kind-argocd-control-plane
+
+# Remove finalizer to allow deletion to complete
+(kubectl get progressivesync -n argocd goinfra -o=json | jq '.metadata.finalizers = null' | kubectl apply -f -) || echo "Continuing.."
```

Comment:
have you tried to use `kubectl patch` here?

Reply:
Done!